### PR TITLE
make toast for ios and android and change width text input

### PR DESCRIPTION
**Pull request by sutani**
#239 change width
#236 add toast

please review my PR

### CODE REVIEW

**Comment by Monte9** [comment]
@sutani Thanks for the PR. I'll review it soon. 

Can you also please attach screenshots of what the component looks like? Thanks.

**Comment by sutani** [comment]
ok @Monte9 
i have attach screenshots

**Comment by Monte9** [comment]
Looks good. This is a great start and I can see Toast being a great addition to React Native Elements. 

With that being said, I think the UI and Component API for Toast can still be improved. 
At RNE the 🔑  for all components is Customization + Polish + Easy of use. So consider flushing the component to handle several props for containerStyling etc. Look at `Button.js` for examples.

Also I wonder what you think of this? https://github.com/react-native-training/react-native-elements/issues/190
Do you think, this could be a good use case for Toasts with gestures?

**Comment by sutani** [comment]
yes, I think it is much better, I made it once so that the base can be used on ios and android, so I'm going to discover again for toast, using message notification bar like #190 @Monte9

File: package.json [hunk 1]
```diff
@@ -23,6 +23,9 @@
   },
   "homepage": "https://github.com/dabit3/React-Native-Elements#readme",
   "dependencies": {
+    "i": "^0.3.5",
```
Comment (binoy14) [inline package.json]: What's the I and npm library needed for?

File: src/form/FormInput.js [hunk 2]
```diff
@@ -127,7 +127,7 @@ styles = StyleSheet.create({
         height: 36,
       }
     }),
-    width: width,
+    width: width * 0.9,
```
Comment (binoy14) [inline src/form/FormInput.js]: What's the reason for this change? It doesn't seem to relate to Toasts?

Comment (Monte9) [inline src/form/FormInput.js]: I believe this is addressing Issue #239. 
I would prefer to have this in a different PR. Also look at [PR 265](https://github.com/react-native-training/react-native-elements/pull/265/files) as this is an attempt to fix the same issue.

**Comment by Monte9** [comment]
I think we will have to move away from this implementation for `Toast`. Here's what we have in mind: https://camo.githubusercontent.com/f6913fda265b52d81f731079425ecb012dca4cbe/68747470733a2f2f7261772e6769746875622e636f6d2f646576427269616e2f72656163742d6e61746976652d64726f70646f776e616c6572742f6d61737465722f73637265656e73686f74732f64656d6f2e676966

I think adding a custom toast component could be more useful and it would help to build it from the ground-up. I don't see the need to depend on an npm package for this. 

@sutani Thoughts? 

/cc @binoy14

**Comment by binoy14** [comment]
Agreed, we want to implement a custom Toast component. Let's try to brain storm some design ideas. The functionality of the Toast component can be a base for the notification component as well.

**Comment by binoy14** [comment]
@sutani Closing this PR because we want to implement a custom component and not use a third part library. Thanks for taking time and effort. We really appreciate your hard work you put into this.
